siteapp/settings.py

```diff
@@ -398,6 +398,9 @@ def make_secret_key():
 	INSTALLED_APPS.append(environment["branding"])
 	TEMPLATES[0].setdefault('DIRS', [])\
 		.insert(0, os.path.join(environment["branding"], 'templates'))
+HEADLESS = False
+if environment.get("headless"):
+	HEADLESS = True
```

Comment:
Not important, but a neat little note: In Python, you can simplify this to one line, like:
`HEADLESS = TRUE if environment.get("headless") else FALSE`

Reply:
Works for me